turf/__init__.py

```diff
@@ -15,6 +15,8 @@
 from turf.length import length
 from turf.midpoint import midpoint
 from turf.nearest_point import nearest_point
+from turf.point_on_feature import point_on_feature
+from turf.nearest_point import nearest_point
```

Comment:
`nearest-point` is being repeated.

Reply:
fixed. this is an artefact of the rebase. My fault.